packages/generator/test/test-util/data-model.ts

```diff
@@ -23,6 +23,9 @@ export const entityName: VdmProperty = {
   isCollection: false
 };
 
+/**
```

Comment:
[req] According to Deeksha's comment this should not be necessary anymore in this file. Can we just remove these comments?

Reply:
done